datafusion/physical-plan/src/aggregates/mod.rs

```diff
@@ -285,7 +285,9 @@ pub struct AggregateExec {
     schema: SchemaRef,
     /// Input schema before any aggregation is applied. For partial aggregate this will be the
     /// same as input.schema() but for the final aggregate it will be the same as the input
-    /// to the partial aggregate
+    /// to the partial aggregate, i.e., partial and final aggregates have same `input_schema`.
```

Comment:
🤔  I went to double check this (being inspired by the thoroughness of your own reviews @viirya ) and I was able to remove the entire field: https://github.com/apache/arrow-datafusion/pull/7741